boards/arm/olimex_stm32_h407/Kconfig.board

```diff
@@ -0,0 +1,11 @@
+# Kconfig - OLIMEX-STM32-H407 board configuration
+#
+# Copyright (c) 2017, Erwin Rol <erwin@erwinrol.com>
```

Comment:
You introduce a new file you don't need to keep this copyright. Please remove it.